srcpkgs/font-adobe-source-code-pro/template

```diff
@@ -1,21 +1,25 @@
 # Template file for 'font-adobe-source-code-pro'
 pkgname=font-adobe-source-code-pro
-version=2.030R
-revision=3
-wrksrc="source-code-pro-${version}-ro-1.050R-it"
+version=2.038R
+revision=1
+wrksrc="source-code-pro-${version}-ro-1.058R-it-1.018R-VAR"
```

Comment:
Change like this make me think it's better to just use `create_wrksrc=yes` then make its own version of `do_extract` to `bsdtar -x --strip-components=1`